examples/langgraph-integration/mcp/README.md

```
@@ -0,0 +1,99 @@
+
+---
+
+### `sequential/sales_analytics/main.py` - Production Sequential Pipeline
+
+Real-world sales analytics workflow with MCP database integration.
+```bash
+make examples-mcp-sales-analytics
+```
+
+**Workflow:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)